src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
```

Comment:
if you add help functions, the code will become clearer
  const findMother = (name) => people.find((p) => p.mother === name);
  const isFemale = (sex) => sex === 'f';